pkg/standalone-utils/test/ProtocolIdRegistry.test.ts

```diff
@@ -99,4 +102,36 @@ describe('ProtocolIdRegistry', () => {
       expect(await registry.isValidProtocolId(MAX_UINT256)).to.equal(false);
     });
   });
+
+  describe('renaming protocol IDs', async () => {
+    const newProtocolName = 'Test Protocol';
```

Comment:
I'd also save the protocol ID under edition as a const here, and use it below.

Reply:
@jubeira What do you mean by this? Isn't it already saved as a const

Reply:
I mean the numeric ID `0`, which is used here and there below.

Reply:
oh gotcha